instrumentation.go

```diff
@@ -401,6 +406,17 @@ func WithEnv() InstrumentationOption {
 
 			err = errors.Join(err, e)
 		}
+		if s, err := newSamplerFromEnv(); err == nil {
+			err := s.validate()
+			if err != nil {
+				return c, err
+			}
+			cfg, err := s.convert()
+			if err != nil {
+				return c, err
```

Comment:
This is dropping all the combined errors from above. The returned errors need to be joined into the already existing errors instead.

Reply:
Right. Updated